### ISSUE TITLE
Plugin isn't reporting anything

### ISSUE THREAD

**Issue by cojoj** [ashfurrow/danger-swiftlint#43]
We're using [buddybuild](http://buddybuild.com/) as CI platform and since also have SwiftLint we've decided that Danger with SwiftLint plugin would be a perfect match to give feedback to PR creators.
I wanted to start simple, so at first I've integrated only Danger and things like:
```ruby
if git.lines_of_code > 500
    warn "Big PR"
end
```
Worked like a charm! ✨

Then I've added `danger-swiftlint` to our Gemfile and used the simplest form in our Dangerfile. To my surprise I didn't get _any_ feedback while SwfitLint run directly reports quite some issues.
I've been trying to modify `.swift` files as well as playing with some options and installation of SwiftLint, but with no luck. TBH, I have no idea where to put my hands now as everything feels pretty much similar to what [firefox](https://github.com/mozilla-mobile/firefox-ios) has.
If you're curious how our implementation looks like, you can take a look at this [Pull Request](https://github.com/JohnSundell/Marathon/pull/106).

**Comment by ashfurrow** [comment]
Hi there – happy to debug this with you! I see you closed the issue – did you get this figured out?

**Comment by cojoj** [comment]
Yeah, I guess so... I’ve played with different swiftlint.yml versions and it feels like we had some conflicts between body lines and file size.

**Comment by ashfurrow** [comment]
Okay, I'm glad 👍 Let me know if you have any other questions.
